lib/gerry/client/request.rb

```diff
@@ -65,7 +65,13 @@ def parse(response)
           raise_request_error(response)
         end
         if response.body
-          JSON.parse(remove_magic_prefix(response.body))
+          source = remove_magic_prefix(response.body)
```

Comment:
Actually, I was just thinking to not call `JSON.parse` at all in this case, but just return the unparsed string. That would have the advantage that we wouldn't have to deal with that bogus single-element array after parsing. What do you think?